asgi_correlation_id/extensions/sentry.py

```diff
@@ -22,7 +22,11 @@ def set_transaction_id(correlation_id: str) -> None:
     The transaction ID is displayed in a Sentry event's detail view,
     which makes it easier to correlate logs to specific events.
     """
-    from sentry_sdk import configure_scope
+    import sentry_sdk
 
-    with configure_scope() as scope:
+    if sentry_sdk.VERSION >= '2.12.0':
```

Comment:
Might want to be careful when comparing strings here:
```pycon
>>> "2.2.0" >= "2.12.0"
True
```

Reply:
thx, already fixed